website/public/tech-guide/index.html

```diff
@@ -0,0 +1,693 @@
+<!doctype html>
+<html>
```

Comment:
**thought:** Outside of scope, but it would be nice if we could pull out the html that's shared across all the files.

Reply:
Ack. Thanks!